utilities_common/multi_asic.py

```diff
@@ -0,0 +1,159 @@
+import subprocess
+import json
+import click
+import functools
+from sonic_device_util import get_num_npus
+from sonic_device_util import get_namespaces
+from sonic_device_util import is_multi_npu
+from sonic_device_util import is_port_internal
+from sonic_device_util import is_port_channel_internal
+from sonic_device_util import is_bgp_session_internal
+from sonic_device_util import get_all_namespaces
+
+import swsssdk
+from swsssdk import ConfigDBConnector
+import argparse
+
+DEFAULT_NAMESPACE = ''
+DISPLAY_ALL = 'all'
+DISPLAY_EXTERNAL = 'frontend'
+PORT_CHANNEL_OBJ = 'PORT_CHANNEL'
+PORT_OBJ = 'PORT'
+BGP_NEIGH_OBJ = 'BGP_NEIGH'
+
+class MultiAsic(object):
```

Comment:
This sounds like the class is for 'MultiAsic' only where as, If I understand correctly, all CLIs will use it on both single ASIC and multi ASIC platforms. Wondering if we should rename to something like 'SwitchAsic'. Either way is fine, lets hear others opinion.

Reply:
I agree the name is a bit misleading, but I'm not sure if `SwitchAsic` is better. I feel like it needs a more generic name, but I can't think of anything at the moment. Maybe as more functionality is added to it we'll have a better idea of what to call it. I'm OK with saving a renaming for a future PR.